Add README.md

dmarmul opened the pull request:
Added README file and some images for describing project

dmarmul committed:
Added README file and some images for describing project

Elena-Bruyako reviewed (changes requested):
add which technologies should I have when clone the project, add steps how to clone the project and run

dmarmul committed:
Added list technologies to get up and running the project, add steps how to clone the project and run.

dmarmul committed:
Changed screenshots to postman link, added aws link.
I'll record video to the second pet project, if its optional. Because, how I understand this project we don't use in the summary.
